{{cookiecutter.package_name}}/noxfile.py

```diff
@@ -115,30 +132,63 @@ def run_shellcheck(session, mode="check"):
     session.run(*shellcheck_cmd, external=True)
 
 
-@nox.session(name="format", python=PYTHON_DEFAULT_VERSION)
+@nox.session(name="format", python=PYTHON_DEFAULT_VERSION, tags=["format", "check"])
 def format_(session):
     """Lint the code and apply fixes in-place whenever possible."""
-    install(session, "lint")
+    install(session, "lint", no_self=True, no_default=True)
     session.run("ruff", "check", "--fix", ".")
     run_shellcheck(session, mode="fmt")
     run_readable(session, mode="fmt")
     session.run("ruff", "format", ".")
 
 
-@nox.session(python=PYTHON_DEFAULT_VERSION)
+@nox.session(python=PYTHON_DEFAULT_VERSION, tags=["lint", "check"])
 def lint(session):
     """Run linters in readonly mode."""
     install(session, "lint")
-    session.run("ruff", "check", "--diff", ".")
+    session.run("ruff", "check", "--diff", "--unsafe-fixes", ".")
+    session.run("ruff", "format", "--diff", ".")
+    session.run("mypy", ".")
     session.run("codespell", ".")
     run_shellcheck(session, mode="check")
     run_readable(session, mode="check")
-    session.run("ruff", "format", "--diff", ".")
-    session.run("bandit", "--ini", "bandit.ini", "-r", ".", *session.posargs)
-    session.run("mypy", "--config-file", "mypy.ini", ".", *session.posargs)
 
 
-@nox.session(python=PYTHON_VERSIONS)
+@nox.session(python=PYTHON_VERSIONS, tags=["test", "check"])
+# COOKIECUTTER{%- if cookiecutter.is_django_package == "y" %}
+@nox.parametrize("django", DJANGO_VERSIONS)
+def test(session, django: str):
+    install(session, "test")
+    session.run("pip", "install", f"django~={django}.0")
+# COOKIECUTTER{%- else %}
 def test(session):
     install(session, "test")
-    session.run("pytest", "-vv", "-n",  "auto", *session.posargs)
+# COOKIECUTTER{%- endif %}
+    session.run("pytest", "-vv", "-n", "auto", *session.posargs)
+
+
+@nox.session(python=PYTHON_DEFAULT_VERSION)
+def make_release(session):
+    parser = argparse.ArgumentParser()
+
+    def version(value):
+        if not re.match(r"\d+\.\d+\.\d+", value):
```

Comment:
It's just a with list item but supporting alpha versions would be useful. I think it was needed in CH.